src/celengine/render.cpp

```diff
@@ -4295,7 +4277,7 @@ Renderer::renderAnnotations(vector<Annotation>::iterator startIter,
     float d0 = farDist - nearDist;
     float d1 = -(farDist + nearDist) / d0; // Used in perspective projection
     float d2 = -2.0f * nearDist * farDist / d0; // Used in perspective projection
-    bool fisheye = projectionMode == ProjectionMode::FisheyeMode;
+    bool fisheye = getProjectionMode()->isFisheye();
```

Comment:
as you provide new classes to handle projection, try to avoid direct `fisheye` usage